src/release/utils/configBuilder.ts

```diff
@@ -51,6 +52,7 @@ export async function buildProjectReleaseConfigs(
   releaseTagManagers: Record<string, ReleaseTagManager>
 ) {
   if (!validateProjectReleaseConfig(configs)) {
+    logger.error({errors : validateProjectReleaseConfig.errors}, 'The config file validation failed');
```

Comment:
How about passing the `logger` to the `new Ajv()` call on line 13 instead?

Reply:
I tried to add the custom logger in the ajv instance like [here](https://ajv.js.org/api.html#error-logging) but when I tried I don't have any logs.

So instead, I implemented the log like the example code [here](https://ajv.js.org/guide/getting-started.html#basic-data-validation)